docs/source/upcoming_release_notes/193-enh_plan_step.rst

```diff
@@ -0,0 +1,24 @@
+193 enh_plan_step
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- Adds backend for running Bluesky plans in active chekckouts
```

Comment:
```suggestion
- Adds backend for running Bluesky plans in active checkouts
```

Reply:
I'm always impressed you have the mental bandwidth to check spelling during code reviews, I don't haha